crates/rpc/src/handlers/blob_sidecar.rs

```diff
@@ -0,0 +1,59 @@
+use actix_web::{
+    HttpResponse, Responder, get,
+    web::{Data, Json, Path},
+};
+use ream_consensus::blob_sidecar::{BlobIdentifier, BlobSidecar};
+use ream_storage::{db::ReamDB, tables::Table};
+use ssz_types::FixedVector;
+use tracing::error;
+
+use crate::{
+    handlers::{
+        block::{get_beacon_block_from_id, get_block_root_from_id},
+        header::get_header_from_slot,
+    },
+    types::{errors::ApiError, id::ID, query::BlobSidecarQuery, response::BeaconVersionedResponse},
+};
+
+#[get("/beacon/blob_sidecars/{block_id}")]
+pub async fn get_blob_sidecars(
+    db: Data<ReamDB>,
+    block_id: Path<ID>,
+    query: Json<BlobSidecarQuery>,
+) -> Result<impl Responder, ApiError> {
+    let block_id = block_id.into_inner();
+    let block_root = get_block_root_from_id(block_id.clone(), &db).await?;
+    let beacon_block = get_beacon_block_from_id(block_id, &db).await?;
```

Comment:
```suggestion
pub async fn get_blob_sidecars(
    db: Data<ReamDB>,
    block_id: Path<ID>,
    query: Json<BlobSidecarQuery>,
) -> Result<impl Responder, ApiError> {
    let beacon_block = get_beacon_block_from_id(block_id.into_inner(), &db).await?;
```

We can derive the block_root from the beacon block so we can avoid these redundant look ups